tests/test_view_model.py

```diff
@@ -0,0 +1,43 @@
+from todo_app.data.classes.item import Item
+from todo_app.data.classes.list import List
+from todo_app.data.classes.viewmodel import ViewModel
+
+import pytest
+
+
+@pytest.fixture
+def view_model():
+    items = [ 
+        Item(1, "Item 1", 1),
+        Item(2, "Item 2", 2),
+        Item(3, "Item 3", 3)
+    ]
+
+    lists = [
+        List(1, 'To Do'),
+        List(2, 'Doing'),
+        List(3, 'Done')
+    ]
+
+    yield ViewModel(items, lists)
+
+def test_view_model_to_do_items_returns_to_do_items(view_model: ViewModel):
+    # Act
+    filtered_items = view_model.to_do_items
+
+    # Assert
+    assert filtered_items == [ view_model._items[0] ]

```

Comment:
this does what you want it to be doing. That said, you are accessing a ViewModel properties to compare them to each other.

What happens when your implementation of ViewModel accidentally changes?
Is it possible that you compare values that are both incorrect such that the test passes?

I suggest you make the expected value clear by initializing a new 
`expected = Item(1, "Item 1", 1)`
comparing it to the value from view model 
`assert filtered_items = [ expected ]`

Because they are objects different objects, you will need to think about how to compare them. Here simply comparing object properties is good enough.
The cleanest way is to implement comparison between them is to implement `def __eq__(self, other): ...` https://stackoverflow.com/questions/1227121/compare-object-instances-for-equality-by-their-attributes

Reply:
note that this is similar in other tests